src/protocols/interaction_model/Constants.h

```diff
@@ -62,6 +62,32 @@ enum class MsgType : uint8_t
     TimedRequest          = 0x0a,
 };
 
+enum class ProtocolCode : uint16_t
```

Comment:
Where are these defined from?

Reply:
they are defined in https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/a9781609d101256455fd7ce52ec2b28e5853a012/src/data_model/Interaction-Model.adoc#status-code-table